test/helpers/kubeclient_helper.rb

```diff
@@ -0,0 +1,21 @@
+module KubeclientHelper
+  def kubeclient
+    @kubeclient ||= begin
+      config = Kubeclient::Config.read(ENV["KUBECONFIG"])
+      unless config.contexts.include?("minikube")
+        raise "`minikube` context must be configured in your KUBECONFIG (#{ENV["KUBECONFIG"]})"
+      end
+
+      client = Kubeclient::Client.new(
+        config.context.api_endpoint,
+        config.context.api_version,
+        {
+          ssl_options: config.context.ssl_options,
+          auth_options: config.context.auth_options
+        }
+      )
+      client.discover
```

Comment:
I'm fine with doing it eagerly here. It seems likely everything we'd be calling would require discovery anyway. Also, it's probably less confusing to have the backtrace (if auth fails for example) point here instead of to some random `get_{thing}` call.